scss/main/partials/_menu.scss

```diff
@@ -65,7 +65,7 @@ Main menu in page header. It is still vertical on mobile.
     margin-bottom: 2rem;
 
     .menu__item a {
-      padding: .3rem;
+      padding: 0.3rem;
```

Comment:
I guess Codeac would prefer quite the opposite.

Reply:
That's why it's here actually :D I just forgot that this is not private repo :D